### PR TITLE
fix: [FRONT-110] TextField padding space change on mobile and desktop screen

### PR DESCRIPTION
**Pull request by umrzoq-toshkentov**
**PR description:**

**Implemented:**

Text

**Checklist:**

- [x] I ran this code locally
- [x] I wrote the necessary tests
- [x] My code follows the [style guidelines](http://bit.ly/sd-web-style-guide)
- [x] I followed the [instructions](http://bit.ly/sd-web-pr) to create a pull request

**JIRA card:**

https://superdispatch.atlassian.net/browse/FRONT-110

**Should know about:**

Desktop:
Screenshot 2023-10-24 at 12 28 56

Mobile:
image

### CODE REVIEW

File: packages/ui/src/text-field/TextFieldOverrides.tsx [hunk 1]
```diff
@@ -98,7 +98,7 @@ export function overrideTextField(theme: SuperDispatchTheme): void {
 
     input: {
       padding: theme.spacing(1.25, 1.5),
-      [sm]: { padding: theme.spacing(1.25, 1.5) },
+      [sm]: { padding: theme.spacing(0.75, 1) },
```
[the remark this answers is not in the document]
Comment (umrzoq-toshkentov) [reply packages/ui/src/text-field/TextFieldOverrides.tsx:101]: `sm` for mobile screen, it is for smaller screen

[the remark this answers is not in the document]
Comment (timurhamitov) [reply packages/ui/src/text-field/TextFieldOverrides.tsx:101]: @umrzoq-toshkentov Are we changing paddings for mobile or desktop screens? I think we wanted to change them for desktop, but use old paddings for mobile screens.

[the remark this answers is not in the document]
Comment (umrzoq-toshkentov) [reply packages/ui/src/text-field/TextFieldOverrides.tsx:101]: @timurhamitov yes I did like you mentioned. For Desktop there is smaller padding and for Mobile screen bigger paddings.

[the remark this answers is not in the document]
Comment (umrzoq-toshkentov) [reply packages/ui/src/text-field/TextFieldOverrides.tsx:101]: @timurhamitov Sorry I missed sm is for Desktop resolution

**Review by timurhamitov — APPROVED** [review]

**Pull request merged**